src/main/java/core/basesyntax/Command.java

```diff
@@ -0,0 +1,130 @@
+                    System.out.println("File " + fileName + " overwritten!");
+                }
+            } else {
+                Files.createFile(filePath);
+                System.out.println("File " + fileName + " created");
+            }
+        } else {
+            System.out.println("Such path doesn't exist!");
+        }
+    }
+
+    public void callHelp() {
+        String[] commands = {"create", "read", "info", "exit"};
+        System.out.println("All available commands: ");
+        for (String command : commands) {
+            callHelp(command);
+        }
+    }
+
+    public void callHelp(String command) {
+        String answer = command + " -> ";
+        switch (command) {
+            case "create":
+                answer += "Creates a text file at the specified path.";
+                break;
+            case "read":
+                answer += "Reads the file at the specified path "
+                        + "and displays the text in the console.";
+                break;
+            case "info":
+                answer += "Displays brief information on the specified file.";
+                break;
+            case "exit":
+                answer += "Program shutdown.";
+                break;
+            default:
+                answer += "Such command doesn't exist.";
+                break;
+        }
+        System.out.println(answer);
+    }
+
+    public void getInfo(String path, String fileName) throws IOException {
+        if (exists(path, fileName)) {
+            Path pathToFile = Paths.get(path + fileName);
+            LocalDateTime date = Instant
+                    .ofEpochMilli(Files.getLastModifiedTime(pathToFile).toMillis())
+                    .atZone(ZoneId.systemDefault()).toLocalDateTime();
+            String text = new String(Files.readAllBytes(pathToFile));
+            String builder = "Number of characters - " + text.length() + " characters" + "\n"
+                    + "Number of lines - " + Files.lines(pathToFile).count() + " lines" + "\n"
+                    + "Number of words - " + text.split(" ").length + " words" + "\n"
+                    + "Date and time of the last change - " + date.format(FORMATTER) + "\n"
+                    + "Size of file - " + Files.size(pathToFile) + " bytes";
+            System.out.println(builder);
+        }
+    }
+
+    public void readFile(String path, String fileName) throws IOException {
+        if (exists(path, fileName)) {
+            System.out.println(new String(Files.readAllBytes(Paths.get(path + fileName))));
+        }
+    }
+
+    private static boolean exists(String path, String fileName) {
+        if (!Files.exists(Paths.get(path)) || !Files.exists(Paths.get(path + fileName))) {
+            System.out.println("Such path or file doesn't exist!");
+            return false;
+        }
+        return true;
+    }
+
+    public void saveText(String[] text) throws IOException {
```

Comment:
methods which you are using only inside class should be private

Reply:
All those methods that public in Command class are called in FileEditor class including method saveText()